src/main/java/edu/kis/powp/factory/SquareFactory.java

```diff
@@ -0,0 +1,18 @@
+package edu.kis.powp.factory;
+
+import edu.kis.powp.command.ComplexCommand;
+import edu.kis.powp.command.DriverCommand;
+import edu.kis.powp.command.OperateToCommand;
+import edu.kis.powp.command.SetPositionCommand;
+
+public class SquareFactory {
+	public static DriverCommand getSquare() {

```

Comment:
Squere and rectangle factories can be easily refactored to Factory Method pattern